File: 2.0/php/me/deleteApiCredential.php
<?php

class ApiClientSample {

    public static function main() {
        $client = \CallFire\Api\DocumentedClient::createClient("account_login", "account_password");
        $request = $client->deleteApiCredential();
        $request->getOperationConfig()->setPathParameters(array("id" => 425836003));
        $result = $client->request($request);
        $json = json_decode($result->getBody());
    }
}

ApiClientSample::main();

File: 2.0/java/me/deleteApiCredential.java
import com.callfire.api.client.CallfireClient;

class ApiClientSample {
    public static void main(String[] args) {
        CallfireClient callfireClient = new CallfireClient("account_login", "account_password");
        callfireClient.meApi().deleteApiCredentials(425836003L);
    }
}
